### ISSUE TITLE
Sourcemaps not present, and giving warning in webpack5

### ISSUE THREAD

**Issue by k-funk** [twilio/twilio-voice.js#147]
## Issue
Installed files include a reference to a sourcemap that is not actually present. The `//# sourceMappingURL=AudioPlayer.js.map` probably shouldn't be there at all, since `AudioPlayer.js` isn't minified anyways. [Looks like someone else ran into this issue as well](https://github.com/twilio/twilio-voice.js/issues/103).

```
> REACT_APP_ENV=development react-scripts start

[HPM] Proxy created: /cmd,/web  -> https://dev.api.hazel.co
(node:23764) [DEP_WEBPACK_DEV_SERVER_ON_AFTER_SETUP_MIDDLEWARE] DeprecationWarning: 'onAfterSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
(Use `node --trace-deprecation ...` to show where the warning was created)
(node:23764) [DEP_WEBPACK_DEV_SERVER_ON_BEFORE_SETUP_MIDDLEWARE] DeprecationWarning: 'onBeforeSetupMiddleware' option is deprecated. Please use the 'setupMiddlewares' option.
Starting the development server...
Compiled with warnings.

Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/AudioPlayer.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/AudioPlayer.js.map'

Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/Deferred.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/Deferred.js.map'

Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/EventTarget.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/EventTarget.js.map'

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.

WARNING in ./node_modules/@twilio/audioplayer/es5/AudioPlayer.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/AudioPlayer.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/AudioPlayer.js.map'

WARNING in ./node_modules/@twilio/audioplayer/es5/Deferred.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/Deferred.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/Deferred.js.map'

WARNING in ./node_modules/@twilio/audioplayer/es5/EventTarget.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/.../node_modules/@twilio/audioplayer/es5/EventTarget.js.map' file: Error: ENOENT: no such file or directory, open '/Users/.../node_modules/@twilio/audioplayer/es5/EventTarget.js.map'

webpack compiled with 3 warnings
```

```
$ node --version
v18.7.0
$ npm list
├── @twilio/voice-sdk@2.1.1
```

## Why this matters to me
Create React App 5 (uses webpack5) outputs warnings from dependencies. In my project, twilio is the only one outputting warnings. IMO CRA5 should allow turning these warnings off, [but alas, that PR seems to be dead](https://github.com/facebook/create-react-app/pull/11752). However, I've been holding on upgrading to CRA5 for 9 months due to this twilio warning.

---

_Originally posted by @k-funk in https://github.com/twilio/twilio-voice.js/issues/79#issuecomment-1207553475_

**Comment by charliesantos** [comment]
Thanks @k-funk for reporting. I submitted an internal ticket to address this on a future release.

**Comment by todorovv995** [comment]
Hi @charliesantos. Are there any updates on this?

**Comment by charliesantos** [comment]
@todorovv995 this has been prioritized and should be fixed in the upcoming months.

**Comment by charliesantos** [comment]
@k-funk can you please try this RC version? https://github.com/twilio/twilio-voice.js/issues/55#issuecomment-1571122310
We're able to see source maps now in our svelte+vite and angular 16 projects.

**Comment by charliesantos** [comment]
SDK source maps work but the AudioPlayer source maps, which is a dependency, are still missing. We will be addressing them soon. Please stay tuned!

**Comment by charliesantos** [comment]
Fixed in [2.6.0-rc2](https://github.com/twilio/twilio-voice.js/releases/tag/2.6.0-rc2)

**Comment by charliesantos** [comment]
We just released 2.6.0 which contains the fix for this issue.
